sources/platform/integrations/actors/integration_ready_actors.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Integration-ready Actors
+description: Learn how to create Actors that are ready to be integrated with other Actors and tasks
+sidebar_position: 1
+slug: /integrations/actors/integration-ready-actors
+---
+
+# Integration-ready Actors
+
+Any Actor can be used in integrations. In order to provide a smooth experience for its users, there are few things to keep in mind.
+
+## General guidelines
+
+If your Actor is supposed to be used as an integration, it will most likely have an input that can be described as two groups of fields. The first group is the "static" part of input - the fields that have the same value whenever the integration is triggered. The second, "dynamic", group are fields that are specific to the triggering event - information from the run or build that triggered the integration.
```

Comment:
```suggestion
If your Actor is supposed to be used as an integration, it will most likely have an input that can be described as two groups of fields. The first group is the "static" part of the input - the fields that have the same value whenever the integration is triggered. The second, "dynamic", group are fields that are specific to the triggering event - information from the run or build that triggered the integration.
```